overlord/devicestate/devicemgr.go

```diff
@@ -1066,6 +1066,58 @@ func (m *DeviceManager) ensureAutoImportAssertions() error {
 	return nil
 }
 
+func (m *DeviceManager) ensureSerialBoundSystemUserAssertionsProcessed() error {
```

Comment:
I think it would be good for this to have a comment describing the situation when this is useful.

Reply:
added comment